lib/foreman/cast.rb

```diff
@@ -0,0 +1,22 @@
+module Foreman
+  module Cast
+    def self.to_bool(value)
+      case value
+
+        when String
```

Comment:
Is rubocop throwing an error here? [It should](https://github.com/bbatsov/ruby-style-guide#indent-when-to-case).